system-apps/web-browser-app/webClient/src/app/browser/browser-window/browser-window.component.scss

```diff
@@ -0,0 +1,41 @@
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+
+  SPDX-License-Identifier: EPL-2.0
+
+  Copyright Contributors to the Zowe Project.
+*/
+
+:host {
+  flex: 1 1 auto;
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+}
+
+iframe {
+  flex: 1 1 auto;
+  border: 0;
+}
+
+.welcome {

```

Comment:
It is safe because Angular adds a unique attribute `_ngcontent-NN` to the element at run-time and limits the scope of the CSS rule using the attribute:
![image](https://user-images.githubusercontent.com/34669449/79416355-e4617380-7fc8-11ea-8ba6-886b7722de4c.png)
![image](https://user-images.githubusercontent.com/34669449/79416371-f2af8f80-7fc8-11ea-993b-9e78a21b1551.png)
Even if you run multiple Web Browsers Angular adds unique attributes to each `iframe`.

Reply:
By the way, how would this CSS rule cause external bleeding? What did you mean?

Reply:
Anyway I added a prefix `zlux-wb-` for all CSS classes used in Web Browser app.